cmd/juju/status/status_test.go

```diff
@@ -1254,6 +1269,10 @@ var statusTests = []testCase{
 								"public-address": "10.0.1.1",
 							},
 						},
+						"endpoint-bindings": M{
+							"server":       "",
```

Comment:
there should be at least one case with a non "" binding, or else we would potentially miss a bug where space was never properly set